src/resource/Resource.h

```diff
@@ -55,34 +55,35 @@ enum class Version {
     Roy = 1,
     Rachael = 2,
     Zhora = 3,
+    Flynn = 4,
     // ...
 };
 
 struct Patch {
-    uint64_t crc;
-    uint32_t index;
-    uintptr_t origData;
+    uint64_t Crc;
+    uint32_t Index;
+    uintptr_t OrigData;
 };
 
 class Resource {
   public:
-    ResourceMgr* resMgr;
-    uint64_t id; // Unique Resource ID
-    ResourceType resType;
-    bool isDirty = false;
-    void* cachedGameAsset = 0; // Conversion to OoT friendly struct cached...
-    std::shared_ptr<File> file;
-    std::vector<Patch> patches;
+    ResourceMgr* ResourceManager;
+    uint64_t Id; // Unique Resource ID
+    ResourceType ResType;
+    bool IsDirty = false;
+    void* CachedGameAsset = 0; // Conversion to OoT friendly struct cached...
+    std::shared_ptr<OtrFile> File;
+    std::vector<Patch> Patches;
     virtual ~Resource();
 };
 
 class ResourceFile {
   public:
-    Endianness endianness;    // 0x00 - Endianness of the file
-    uint32_t resourceType;    // 0x01 - 4 byte MAGIC
-    Version version;          // 0x05 - Based on Ship release numbers
-    uint64_t id;              // 0x09 - Unique Resource ID
-    uint32_t resourceVersion; // 0x11 - Resource Minor Version Number
+    Endianness ByteOrder;     // 0x00 - Endianness of the file
+    uint32_t ResourceType;    // 0x01 - 4 byte MAGIC
+    Version Ver;              // 0x05 - Based on Ship release numbers
```

Comment:
not the biggest fan of `Ver` as a name here but i can't think of a better one that doesn't make it confusing like `Version` would at the moment

Reply:
Can't be `Version` because gcc can't differentiate between the type and the variable name.